sample/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+add_executable ( ${PROJECT_NAME} main.cpp )
+
+target_link_libraries ( ${PROJECT_NAME} radproto )
+
+target_link_libraries ( ${PROJECT_NAME} OpenSSL::Crypto Boost::boost Threads::Threads )
+
+target_include_directories ( ${PROJECT_NAME} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/../src )
```

Comment:
It is better not to deal with relative paths and use `CMAKE_SOURCE_DIR`, which points to the project root.